ramlfications/loader.py

```diff
@@ -28,7 +28,7 @@ def _yaml_include(self, loader, node):
         file_name = os.path.join(os.path.dirname(loader.name), node.value)
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile)
+            return yaml.load(inputfile, self._ordered_loader)
```

Comment:
This got a lot simpler, since the `OrderedLoader` is based on `yaml.SafeLoader` already. But we have to send it to the `OrderedLoader` or else it breaks the recursion.

Maybe that's a sign that we should change how `OrderedLoader` is created?